### PR TITLE
chore(igxGrid): add known grid limitations section

### LINKED ISSUE #439: Document all known issues and limitation when sizes are set with % or parent sizes are not set at all
In the main grid topic we should add a section for issues and limitation when it comes to set column sizes to be in percentages %.

- [x] One of the explanations should be for the scenario when column widths are set with %
- [x] Another one when columnWidth is set with %
- [x] When grid height is not set, and it is nested in parent without sizes as well. [Example](https://stackblitz.com/edit/angular-pmxpyk-zlnhge)

### PR DESCRIPTION
**Pull request by Aleksandyr**
Closes #439

### CODE REVIEW

File: en/components/grid.md [hunk 1]
```diff
@@ -639,3 +638,17 @@ Our community is active and always welcoming to new ideas.
...
+## Grid Limitations
+
+<div ckass="devider"></div>
+
+### General
+
+|Limitation|Description|
+|--- |--- |--- |
+|Column widths into `percentage`|Currently we do not support mixing column widths with `%` and `px`.
+|Skipped `height` on the parent of the grid or grid itself| The `grid` is not rendered correctly whenever the `height` on the parent or on the grid itself is omitted.|
```
Comment (zdrawku) [inline en/components/grid.md]: When `height` of the grid's parent element or grid itself is not set | The `grid` height is not rendered correctly whenever the `height` of the grid's parent element or to the grid itself, is omitted.

File: en/components/grid.md [hunk 2]
```diff
@@ -639,3 +638,17 @@ Our community is active and always welcoming to new ideas.
...
+|`NaN` whenever filter number column|When filter number column and enter character different from digit, `NaN` value appers due incorrect cast and no filter is applied.|
```
Comment (zdrawku) [inline en/components/grid.md]: `NaN` is returned when filtering columns of type number | If value type different than number is entered in the filtering input, `NaN` value appers due incorrect cast and no filter is applied.

File: en/components/grid.md [hunk 3]
```diff
@@ -639,3 +638,17 @@ Our community is active and always welcoming to new ideas.
...
+|`undefined` in edit mode|When `data` persists `undefined` values and enter into `edit mode` the string value `undefined` is displayed.|
```
Comment (zdrawku) [inline en/components/grid.md]: If the data passed to the grid is having `undefined` field values, the string `undefined` will appear in the cell when it is in edit mode.

File: en/components/grid.md [hunk 4]
```diff
@@ -639,3 +638,17 @@ Our community is active and always welcoming to new ideas.
...
+|Grid `width` |Currently the `width` of the grid is spanning the container(parent) it is placed in. Column `widths` does not reflect on the `view port` of the grid.|
```
Comment (zdrawku) [inline en/components/grid.md]: Grid `width` does not depend on the column widths | The width of all columns does not determine the spaning of the grid itself, it is determined by the parent container dimensions or the difined grid's width.

Grid nested in parent container | When grid's width is not set and it is placed in parent container with defined dimensions, the grid spans to this container.

File: en/components/grid.md [hunk 5]
```diff
@@ -639,3 +638,16 @@ Our community is active and always welcoming to new ideas.
 
 * [Ignite UI for Angular **Forums**](https://www.infragistics.com/community/forums/f/ignite-ui-for-angular)
 * [Ignite UI for Angular **GitHub**](https://github.com/IgniteUI/igniteui-angular)
+
+## Grid Limitations
+
+<div ckass="devider"></div>
```
Comment (ig-ealbert) [inline en/components/grid.md]: Class is misspelled and I'm not sure if the class name is spelled properly either. :)

File: en/components/grid.md [hunk 6]
```diff
@@ -639,3 +638,16 @@ Our community is active and always welcoming to new ideas.
...
+
+|Limitation|Description|
+|--- |--- |
+|Column widths set in `percentage` and `px`|Currently we do not support mixing of column widths with `%` and `px`.
+|When `height` of the grid's parent element is not set| The `grid` is not rendered correctly when the `height` of the parent container is not set.|
+|When grid's `height` is not set|When grid's `height` is not set, it will span the parent cantainer it is placed in.|
```
Comment (ig-ealbert) [inline en/components/grid.md]: container

File: en/components/grid.md [hunk 7]
```diff
@@ -639,3 +638,16 @@ Our community is active and always welcoming to new ideas.
...
+|`NaN` is returned when try to filter column of type number|if value different than `number` is entered into the filtering input, `NaN` is returned due incorrect cast and no filter is applied.|
```
Comment (ig-ealbert) [inline en/components/grid.md]: when trying to filter a column of type `number`
If a value different than `number` is entered into the filtering input, `NaN` is returned due to an incorrect cast

File: en/components/grid.md [hunk 8]
```diff
@@ -639,3 +638,16 @@ Our community is active and always welcoming to new ideas.
...
+|Grid `width` does not depend on the column widths | The `width` of all columns does not determine the spanning of the grid itself, it is determined by the parent container dimensions or the difined grid's `width`.
```
Comment (ig-ealbert) [inline en/components/grid.md]: grid itself.  It is determined by the parent container dimensions or the defined grid's `width`.

File: en/components/grid.md [hunk 9]
```diff
@@ -639,3 +638,16 @@ Our community is active and always welcoming to new ideas.
...
+|Grid nested in parent container | When grid's `width` is not set and it is placed in parent container with defined dimensions, the grid spans to this container.
```
Comment (ig-ealbert) [inline en/components/grid.md]: placed in a parent container

File: en/components/grid.md [hunk 10]
```diff
@@ -639,3 +638,16 @@ Our community is active and always welcoming to new ideas.
...
+
+<div class="devider"></div>
+
+|Limitation|Description|
+|--- |--- |
+|Column widths set in `percentage` and `px`|Currently we do not support mixing of column widths with `%` and `px`.
+|When `height` of the grid's parent element is not set| The `grid` is not rendered correctly when the `height` of the parent container is not set.|
+|When grid's `height` is not set|When grid's `height` is not set, it will span the parent container it is placed in.|
+|When trying to filter a column of type `number`|If a value different than `number` is entered into the filtering input, `NaN` is returned due to an incorrect cast|
+|Grid `width` does not depend on the column widths | The `width` of all columns does not determine the spanning of the grid itself. It is determined by the parent container dimensions or the difined grid's `width`.
```
Comment (zdrawku) [inline en/components/grid.md]: container dimensions or the **defined** grid's

File: jp/components/grid.md [hunk 11]
```diff
@@ -639,3 +639,17 @@ import {
...
+
+<div class="devider"></div>
+
+|Limitation|Description|
+|--- |--- |
+|Column widths set in `percentage` and `px`|Currently we do not support mixing of column widths with `%` and `px`.
+|When `height` of the grid's parent element is not set| The `grid` is not rendered correctly when the `height` of the parent container is not set.|
+|When grid's `height` is not set|When grid's `height` is not set, it will span the parent container it is placed in.|
+|When trying to filter a column of type `number`|If a value different than `number` is entered into the filtering input, `NaN` is returned due to an incorrect cast|
+|Grid `width` does not depend on the column widths | The `width` of all columns does not determine the spanning of the grid itself. It is determined by the parent container dimensions or the difined grid's `width`.
```
Comment (zdrawku) [inline jp/components/grid.md]: container dimensions or the **defined** grid's

**Comment by Aleksandyr** [comment]
@zdrawku @ig-ealbert @yoshimis I'm going to add some new changes I hope that the localization hasn't started yet.

**Comment by ig-ealbert** [comment]
@zdrawku Everything looks ok except the class name for the "devider".  Is it really called that?  Or is it "divider"?

**Comment by zdrawku** [comment]
Good catch @ig-ealbert, it should be divider :) @Aleksandyr

**Comment by zdrawku** [comment]
@yoshimis I am not going to merge this PR, I am going to wait for your translation, you can use the PR to make commits to it, instead of creating a new PR.

**Pull request merged**